add a note that audit log entries may have had values masked. (#2859)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/audit-logs.mdx

  </APIField>
  <APIField name="auditLog.newValue" type="String">
    The new value of a changed object.

>     Starting in version 1.49.0 some sensitive values may have had their values masked.

Reviewer: ```suggestion
    In versions >= 1.49.0 sensitive values may be masked.
```

Would suggest that phrasing.
